.travis.yml

```diff
@@ -0,0 +1,29 @@
+sudo: false
+language: python
+
+matrix:
+  fast_finish: true
+  include:
+    - {python: 2.7, env: test_group=pep8}
+    - {python: 3.6, env: test_group=pep8}
+    - {python: 2.7}
+    - {python: 3.3}
```

Comment:
do we care about 3.3 which is already EOL last September?

Reply:
Basically, the only 3.3 specific part is `SSLContext.check_hostname/match_hostname`
as we've found out in incredible work #90 

I think it's cheap enough to leave for a while.

 https://github.com/httplib2/httplib2/pull/90/files#diff-72a22fe2efc9771bbe023ba1ffa07173R146

Reply:
SGTM